tailormap-components/projects/core/src/lib/user-interface/attributelist/attributelist-panel/attributelist-panel.component.html

```diff
@@ -35,7 +35,7 @@
                    (selectedTabChange)="onSelectedTabChange();">
       <mat-tab *ngFor="let layer of layers; let index = index; trackBy: trackByLayerId" [label]="layer.name">
         <ng-template mat-tab-label>
-          {{layer.name}}
+          {{layer.alias}}
```

Comment:
alias can be empty

Reply:
Ik zal een functie maken die of de alias terug geeft of als deze leeg is, de name.